.gitlab-ci.yml

```diff
@@ -8,14 +8,17 @@ variables:
   PROJECT_ALLOC_NAME: ${CI_PROJECT_NAME}_ci_${CI_PIPELINE_ID}
   BUILD_ROOT: ${CI_PROJECT_DIR}
   ACCOUNT: -p pdebug -A eng
+  L_ACCOUNT: -q pdebug -G eng
   MIRROR: "/usr/workspace/smithdev/mirror"
 
 # There are no tests for now
 stages:
-  - allocate_resources
-  - build
-  - release_resources
-  - build_with_deps
+  - q_allocate_resources
+  - q_build
+  - q_release_resources
+  - q_build_with_deps
+  - l_build
+  - l_build_with_deps
```

Comment:
I like it when someone looked at my work and tries not to stick to some conventions. It’s a detail, but I can tell from this that you did so, and I’m grateful.